tests/sdk/testTransaction.py

```diff
@@ -42,6 +43,15 @@ def test_success(self):
         self.assertIsNotNone(transaction.id)
         self.assertEqual(transaction.id, transaction_id)
 
+    def test_success_ids(self):
+        transactions = starkbank.transaction.query(limit=5)
+        transactions_ids_expected = [t.id for t in transactions]
+        transactions_ids_result = [t.id for t in starkbank.transaction.query(ids=transactions_ids_expected)]
+        transactions_ids_expected.sort()
+        transactions_ids_result.sort()
+        self.assertIsNotNone(transactions_ids_result)
```

Comment:
Me parece que transactions_ids_result pode ser uma lista vazia, mas não None

Reply:
Changed to assertTrue.